app/controllers/admin/application_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Admin
+  class ApplicationController < ActionController::Base
+    before_action :reject_unless_admin, :authenticate_user!
+    before_action :configure_permitted_parameters, if: :devise_controller?
+    layout 'admin/layouts/application'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.